backend/chainlit/langchain/callbacks.py

```diff
@@ -587,12 +587,13 @@ def _on_run_update(self, run: Run) -> None:
         outputs = run.outputs or {}
         output_keys = list(outputs.keys())
         output = outputs
+
         if output_keys:
             output = outputs.get(output_keys[0], outputs)
-
+            
         if current_step:
             current_step.output = (
-                output[0] if isinstance(output, Sequence) and len(output) else output
+                output[0] if isinstance(output, Sequence) and not isinstance(output, str) and len(output) else output
```

Comment:
Since a string is a `Sequence`, it would only take the first character of the string. This is improving the type check to make sure the output is correctly logged.